### ISSUE TITLE
Full page alert banner - Focus not returned to top of page when alert is closed

### ISSUE THREAD

**Issue by sgroi-l** [localgovdrupal/localgov_alert_banner#377]
### Description

When closing an alert on the webpage, focus is not returned to the top of the page. This issue is present when using keyboard navigation (tabbing) or clicking 'Close Dialog' in Chrome. 

### Expected Behaviour

Upon closing the alert (either by tabbing or clicking 'Close'), focus should be returned to the top of the page. The next focus item should be 'Skip to main content' 

![image](https://github.com/user-attachments/assets/0a0e718c-cad8-49d2-9174-028ff2f665fe)

### Actual Behaviour

- **Firefox (Clicking 'Close')**: Focus is returned to the top of the page, as expected.
- **Firefox (Tabbing to Close)**: Focus is not returned to the top of the page.
- **Chrome (Both Clicking and Tabbing to Close)**: Focus is not returned to the top of the page.

### How to Replicate

1. Visit the page where the alert is automatically displayed.
2. Close the alert by:
   - Clicking the 'Close' button.
   - Using keyboard navigation to reach the 'Close' button.
3. Press tab to observe whether the focus is taken back to the top of the page.

### WCAG Reference

- **WCAG Guideline 2.4.3: Focus Order (A)**  
  *If a web page can be navigated sequentially and the navigation sequence affects meaning or operation, focusable components should receive focus in an order that preserves meaning and operability.*

More details: [WCAG 2.4.3 Focus Order](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html)

**Comment by markconroy** [comment]
This is an interesting one. As we are using the `dialog` element, focus should return to the item that triggered the dialog to show. However, in our case nothing is triggering the dialog since it's a full-page takeover and is taking over the page upon load.

It's easy to set the focus back to Skip to Content but that's a hidden item that should only be visible when explicitly focussed. I'm not fully convinced yet that it's the right place to put our focus. At the same time, I can't think of somewhere better either.

I'd like to figure out why the page is deciding to focus where it does and work back from there to what we _should_ do.
